src/legacy/core_plugins/region_map/public/region_map_vis_params.js

```diff
@@ -66,21 +66,6 @@ uiModules.get('kibana/region_map')
               setTimeout(function () {
                 $scope.dirty = false;
               }, 0);
-              $scope.collections.vectorLayers = newVectorLayers;
```

Comment:
It is not immediately clear to me what this is being removed for - how is it related to the other params updates?

Reply:
its duplicated code, not really related to all this, but my exploration started in the regionmap, i found this duplicate code and removed it. If you prefer i can take it out of this PR.

Reply:
cool, not too worried about it being included here, just wanted to understand what it was for